bugbug/bugzilla.py

```diff
@@ -93,50 +94,84 @@ def bughandler(bug):
     return all_ids
 
 
-def get(ids_or_query):
-    new_bugs = {}
+def get(bug_ids: Union[List[int], Set[int]]):
```

Comment:
You need to add the return type too

Reply:
Resolved by adding `Dict[int, dict]` as return type